splinterd/tests/framework/network.rs

```diff
@@ -44,28 +45,38 @@ impl Network {
             nodes: Vec::new(),
             temp_dirs: HashMap::new(),
             external_registries: None,
+            num_of_keys: 1,
         }
     }
 
     pub fn add_nodes_with_defaults(mut self, count: i32) -> Result<Network, InternalError> {
         let mut registry_info = vec![];
         let context = Secp256k1Context::new();
         for _ in 0..count {
-            let signer = context.new_signer(context.new_random_private_key());
-            let public_key = signer
+            let admin_signer = context.new_signer(context.new_random_private_key());
+            let public_key = admin_signer
                 .public_key()
                 .map_err(|e| InternalError::from_source(Box::new(e)))?;
             let temp_dir = TempDir::new("scabbard_data")
                 .map_err(|e| InternalError::from_source(Box::new(e)))?;
 
+            let mut signers = vec![admin_signer.clone()];
```

Comment:
Is the admin signer intended to be used in the same way as the other signers?

Reply:
That was a hold over from the first challenge authorization test that was added. With the addition of supporting multiple keys, I updated the commit to remove the admin signer from the signers list.